spec/features/building_map_spec.rb

```diff
@@ -78,7 +78,7 @@
     expect(page).not_to have_css(".pin-icon2")
   end
 
-  it "removes a pin when clicked again", js: true do
+  it "removes a pin when clicked again", js: true, inconsistent: true, local_only: true do
```

Comment:
Are these changes supposed to be in this PR?

Reply:
They shouldn't but otherwise these tests may fail. In the best case we can commit without these changes but last time i committed without them a test failed...
Will get fixed anyway with #210

Reply:
now they work